.github/workflows/ci.yml

```
@@ -11,25 +11,14 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+      - name: Install pnpm
+        uses: pnpm/action-setup@v4
       - name: Using Node.js from .nvmrc
         uses: actions/setup-node@v3
```

actionlint (1.7.12)

[error] the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)